bin/run.js

```diff
@@ -27,6 +27,12 @@ try {
 }
 
 // process runfile.js
-console.log('Processing runfile...')
-var runfile = require(path.resolve('./runfile'));
+try {
+    console.log('Processing runfile...')
+    var runfile = require(path.resolve('./runfile'));
```

Comment:
Hmm true. _RegExp_ is definitely an overkill. Let's just do a simple check if file exist before requiring. `fs.accessSync()` ? Simple and clear. What do you think ?